### DISCUSSION TITLE
Best practices for syncing multiple desktops

### DISCUSSION THREAD

**Discussion by kyouens** [scambier/obsidian-omnisearch#255] [Q&A] [answered]
Hi,
Love the plugin.  I have multiple desktops that use Obsidian Sync.  I want to use Omnisearch on all of them.  Should I enable PDF and image indexing on all of them, or just on one of them? Are there any best practices to avoid issues in this setting?  Thanks so much for your work on this--great addition to Obsidian.
Kenny

**Comment by scambier** [comment, marked as answer]
You need to enable PDF/Image indexing on all devices where you need the feature - so I guess _all_ devices. **You also want to make sure that Obsidian Sync correctly syncs Text Extractor's cache**, to avoid redundant work as much as possible. The cache files will (hopefully) get synced fast enough, and Text Extractor won't do the work twice.

Edit: The caches files are located in `.obsidian\plugins\text-extractor\cache`

To avoid conflicts between your desktops, it might be safer to let Text Extractor do its work on a single machine _and then_ sync with the others.

**Comment by kyouens** [comment]
Thank you! That’s exactly the info I needed. Much appreciated.
________________________________
